e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -138,7 +145,8 @@ suite('Validation of workspace build and run', async () => {
     });
 
     test('Check the running application', async () => {
-        await previewWidget.waitContentAvailable(SpringAppLocators.springTitleLocator, 60000, 10000);
+        await previewWidget.waitContentAvailableInAssoziatedWorkspace(SpringAppLocators.springTitleLocator, 60000, 10000);
+        await driverHelper.getDriver().switchTo().defaultContent();
```

Comment:
move this inside of "waitContentAvailableInAssoziatedWorkspace" method

Reply:
typo was fixed